### ISSUE TITLE
Using `create-bucket` and `--lifecycleRules` correctly

### ISSUE THREAD

**Issue by anjos** [Backblaze/B2_Command_Line_Tool#432]
I'm trying to use this flag on my system to create a bucket in which only the last version of the files are kept, but I have found no way to pass the correct set of parameters that can be accepted or documentation with examples. Here is what I'm trying:

```sh
$ b2 create-bucket --lifecycleRules '{"daysFromHidingToDeleting": 1, "daysFromUploadingToHiding": null, "fileNamePrefix": ""}' my-very-own-test-bucket allPrivate
ERROR: Bad request: array should start with bracket but found: {
```

I'm confused as it is stated here it should be a dictionary: https://www.backblaze.com/b2/docs/lifecycle_rules.html

Could somebody show a working example of this flag?

**Comment by anjos** [comment]
There can be many rules per bucket - the command expects:

```sh
$ b2 create-bucket --lifecycleRules '[{"daysFromHidingToDeleting": 1, "daysFromUploadingToHiding": null, "fileNamePrefix": ""}]' my-very-own-test-bucket allPrivate
```

It would be interesting to add this to the documentation though.

**Comment by mlech-reef** [comment]
The error message should be improved by Backblaze/b2-sdk-python#188

**Comment by mjurbanski-reef** [comment]
resolved with [4.0.2](https://github.com/Backblaze/B2_Command_Line_Tool/releases/tag/v4.0.2)
